tests/Builder/var-saved/build_config.json

```diff
@@ -1,18 +1,22 @@
 {
-    "checksum": "d5d5e0eb56c03942794d09e8303acc93",
+    "checksum": "942d0cebe2044bc379a3b3d08584ac56",
     "mapping": {
         ".js": ".js"
     },
     "paths": {
-        "root": "",
+        "root": "\/home\/nutama\/github\/asset-lib\/tests\/Builder\/",
```

Comment:
maybe not have abslute paths? same for all those in this file

Reply:
I can not reproduce how this files was changed.
Removing and regenerating the file does creates an incorrect file, however, I am certain that I did not do that.
Reverted the changes in the file.